styles/components/buttons/_config.scss

```diff
@@ -8,62 +8,62 @@ $color-g1: var.$color-grey-1 !default;
 $color-g2: var.$color-grey-2 !default;
 $color-g3: var.$color-grey-3 !default;
 $color-g4: var.$color-grey-4 !default;
-$color-g6: var.$color-grey-6 !default;
+$color-g5: var.$color-grey-5 !default;
```

Comment:
The changes in this module are just updating internal variable names. No visible changes—resulting colors are the same.